seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/ClusterFaultToleranceTwoPipelineIT.java

```diff
@@ -219,7 +209,7 @@ public void testTwoPipelineStreamJobRunOkIn3Node() throws ExecutionException, In
                 return clientJobProxy.waitForJobComplete();
             });
 
-            Awaitility.await().atMost(3, TimeUnit.MINUTES)
+            Awaitility.await().atMost(200000, TimeUnit.MINUTES)
```

Comment:
2000 minutes is too long, it will lead to a long wait.

Reply:
and hide errors

Reply:
done